### PR TITLE
[CLA] Agile Business Group

### PR DESCRIPTION
**Pull request by tafaRU**

### CODE REVIEW

**Comment by mart-e** [comment]
Thanks, if you are adding them here, no need to keep the individuals, you can remove them.
I already have an individual CLA for @hurrinico, @SimoRubi and @SimoneVagile as employees of Agilebg

**Comment by tafaRU** [comment]
I'd keep mine as I'm fond of it :blush: 

>  I already have an individual CLA for @hurrinico, @SimoRubi and @SimoneVagile as employees of Agilebg

They don't work anymore in Agile BG: could you prefer that include removing of their corresponding files in only one commit? Same as I added the corporate cla.

**Comment by mart-e** [comment]
What you can do is add the date of validity next to the email. Example:
https://raw.githubusercontent.com/odoo/odoo/15.0/doc/cla/individual/eLBati.md

**Comment by tafaRU** [comment]
> What you can do is add the date of validity next to the email.

Do you mean for the ex-employees?

**Comment by mart-e** [comment]
Yes, so you only have active employees in the corporate one :)

**Comment by tafaRU** [comment]
@mart-e done

File: doc/cla/individual/tafaRU.md [hunk 1]
```diff
@@ -8,4 +8,4 @@ declaration.
 
 Signed,
 
-Alex Comba alex.comba@agilebg.com https://github.com/tafaRU
+Alex Comba alex.comba@agilebg.com (up to 2022-08-26) https://github.com/tafaRU
```
Comment (mart-e) [inline doc/cla/individual/tafaRU.md:11]: Sorry if I was not clear. The `up to <date>` is just a convention we use to say the CLA is no longer valid past this date. You can write it next to the CLA of the users that should no longer commit using this email address (so the ones no longer working for your company). So don't write it next to the emails of the ones still working at agilebg.

Comment (tafaRU) [reply doc/cla/individual/tafaRU.md]: Ok no problem, I should have fixed.

**Comment by mart-e** [comment]
Hello,

Thank you for the signature, it will soon be integrated into Odoo.
Future pull requests made on odoo will be validated by the runbot.
If you have existing PRs that are awaiting for CLA validation, don't forget to rebase your code as soon as this is merged.

Regards

@robodoo review+

**Pull request closed without merging**